lesson1.html

```diff
@@ -229,7 +229,7 @@ <h2>Whiteboard Exercise</h2>
         <section>
           <h2>Whiteboard Exercise</h2>
           <p>Write pseudocode for the following:</p>
-          <blockquote>Santa doesn’t know how many houses are on Long Street but he knows he’ll get too tired if he visits too many. He will keep visiting houses until he’s visited 108 in total.</blockquote>
+          <blockquote>Hotels.com is running a giveaway competition and wants to give away as many prizes as possible with a $500,000 budget. Keep giving away prizes until this budget has been used up.</blockquote>
```

Comment:
I think we need to specify a prize amount or say it can be specified by Hotels.com